nx-X11/programs/Xserver/hw/nxagent/NXdamage.c

```diff
@@ -1,5 +1,2073 @@
-#ifdef NXAGENT_UPGRADE
```

Comment:
OK, merging `nx-X11/programs/Xserver/hw/nxagent/NXdamage.c` and `nx-X11/programs/Xserver/hw/nxagent/X/NXdamage.c` sounds sane.